From 292db5db488b2f89a3b8c3ad7aac6e2606ff7cd8 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Tue, 6 Dec 2011 17:24:29 +0800
Subject: [PATCH 0709/2508] ENGR00162181-2 GPMI : select the MXS-DMA and
 MTD_CHAR

select the MXS-DMA and MTD_CHAR macros.
The gpmi driver needs them to be enabled.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 drivers/mtd/nand/Kconfig |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/mtd/nand/Kconfig b/drivers/mtd/nand/Kconfig
index 01ccf52..53d9906 100644
--- a/drivers/mtd/nand/Kconfig
+++ b/drivers/mtd/nand/Kconfig
@@ -428,6 +428,8 @@ config MTD_NAND_GPMI_NFC
         depends on MTD_NAND && (SOC_IMX23 || SOC_IMX28 || SOC_IMX50 || SOC_IMX6Q)
 	select MTD_PARTITIONS
 	select MTD_CMDLINE_PARTS
+	select MTC_CHAR
+	select MXS_DMA
         help
 	 Enables NAND Flash support for IMX23 or IMX28.
 	 The GPMI controller is very powerful, with the help of BCH
-- 
1.7.7.4

